exalearn-api/ExaLearn.WebApi/appsettings.json

```diff
@@ -8,7 +8,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DbContext": "Server=localhost;Port=5432;Username=postgres;Password=;Database=exalern_db",
+    "DbContext": "Server=localhost;Port=5432;Username=postgres;Password=Pass12345;Database=exalern_db",
     "HostingContext": "linkToFilesHosting"
```

Comment:
remove your local settings for Password field

Reply:
this password is required to connect to the server. if you need access to the local database, just change it.

Reply:
that why you must to create new file appsettings.exalearn.ru.json and must to move settings of server to this file, the expected result will be two files : first file with local setting and second file with server settings